database/migrations/2023_07_26_135328_change_default_exchange_rate_on_projects_table.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('projects', function (Blueprint $table) {
+            $table->decimal('exchange_rate')->nullable(false)->default(1)->change();
+        });
```

Comment:
When trying this out, this migration failed for me because I had existing exchange rates that were null. it's a shame it doesn't seem to automatically set those to the new default value, but I needed to add in an update statement first to make this work.